util/printchplenv

```diff
@@ -93,7 +93,7 @@ def print_mode(mode='list', anonymize=False):
 
     atomics = chpl_atomics.get()
     print_var('CHPL_ATOMICS', atomics, mode, 'atomics', ('runtime', 'launcher'))
-    if (mode != 'runtime' and mode != 'make' and mode != 'launcher') and comm != 'none' or mode == 'simple':
+    if (mode != 'runtime' and mode != 'make' and mode != 'launcher') and comm != 'none' or mode == 'simple' or mode == 'shell':
```

Comment:
This should use 'm' instead of 'mode' so that csh and sh will print the same set of variables.